docs/HardwareReservationsApi.md

```diff
@@ -168,6 +168,7 @@ Name | Type | Description  | Notes
 Retrieve all hardware reservations for a given project
 
 
+FindProjectHardwareReservations is a paginated API operation. You can specify page number and number of hardwarereservations per page in parameters, or you can call `ExecuteWithPagination()` to fetch all the hardwarereservations.
```

Comment:
there is no mustache lambda in openapi generator that would break camelcased words :sweat_smile:

Reply:
I'll comment on the template